docs/getting-started/installation.md

```diff
@@ -5,143 +5,133 @@ icon: material/cogs
 
 # Installation
 
+!!! danger "Removed Features"
+	Before installing, it is highly recommended to take a look at our [Removed Features](/troubleshooting/removed-features/) page.
+
 !!! info
     Atlas no longer provides ISOs to download, please use this method here to install.
 
 ## :material-clipboard-list-outline: Requirements
 
-* A USB drive (8 GB minimum)
-* Bootable USB creator
-* A backup of all the files that you want to keep, either in the cloud or on an external drive.
 * [AME Wizard](https://atlasos.net/downloads)
 * [Atlas Playbook](https://atlasos.net/downloads)
-* x64/64-bit CPU (x86 or 32-bit is not supported)
+* x86_64/64-bit CPU (x86 or 32-bit is not supported)
```

Comment:
Might this cause confusion to some less knowledgeable? Saying "x86_64" and then stating x86 is unsupported.
Would it maybe be better to just say 64-bit and provide short guidance on how to check?

Reply:
Probably, x64 is a different architecture so it was a correction. I'll change to just 64-bit.